app/grandchallenge/challenges/templates/challenges/challenge_list.html

```diff
@@ -17,7 +17,7 @@
             </a>
         </div>
         <div class="col-xl-4 col-lg-5 col-md-6 mb-3 text-md-right">
-            {% if perms.challenges.change_challengerequest %}
+            {% if perms.challenges.view_challengerequest %}

```

Comment:
This should probably still be change right? Or we could remove the button altogether as its now in the nav.

Reply:
well both the global view and the global change permissions are only given to reviewers, so doesn't really matter.

Reply:
I think get rid of the button as it's no longer needed.